### ISSUE TITLE
Remove cruft

### ISSUE THREAD

**Issue by rpruim** [ProjectMOSAIC/mosaicModel#3]
@dtkaplan, are there are any things in this package that we don't need.  I think I got rid of the `ggformula` stuff, but there may be other things that were build along that way that aren't really needed or don't belong in this package.  You probably know where to look better than anyone.

**Comment by dtkaplan** [comment]
@rpruim, Yes. I'm starting to go through this now. I'll do so in a branch named issue003. I'll also put in the `mod_` series of names.

**Comment by rpruim** [comment]
@dtkaplan, is this closeable now?
